hybrasyl/Objects/VisibleObject.cs

```diff
@@ -82,8 +82,8 @@ public bool CanBeLooted(string username, out string error)
             if (DeathPileTime == null) return true;
             if (DeathPileOwner == username) return true;
             if (DeathPileAllowedLooters.Contains(username) &&
-                (DateTime.Now - DeathPileTime.Value).Seconds > Constants.DEATHPILE_GROUP_TIMEOUT) return true;
-            if ((DateTime.Now - DeathPileTime.Value).Seconds > Constants.DEATHPILE_RANDO_TIMEOUT) return true;
+                (DateTime.Now - DeathPileTime.Value).TotalSeconds > Constants.DEATHPILE_GROUP_TIMEOUT) return true;
```

Comment:
Is this related to looting items from mobs? If so, deathpiletime should not be used. New functionality should be created to handle looting versus death pile of a user.

Reply:
Okay. Will look into that.